udata_hydra/analysis/csv.py

```diff
@@ -99,9 +99,9 @@ async def notify_udata(check_id):
 
 
 async def analyse_csv(
-    check_id: int = None,
-    url: str = None,
-    file_path: str = None,
+    check_id: int | None = None,
+    url: Union[str, None] = None,
```

Comment:
Why use the `Union[str, None]` version, and not its shorthand (which seems to [be recommended](https://docs.python.org/3/library/typing.html#typing.Union)) `str | None`?

I'm new to python typing annotations, so I may be missing something obvious here 😬

Reply:
You're totally right, but we decided with @maudetes that we would keep hydra as Python 3.9 compatible first (and "|" is not yet comaptible with Python 3.9), and we will later move to a nicer Python>=3.10 syntax!

Reply:
FYI there is another branch to fully switch to Python 3.11 including nicer typing not compatible with 3.9, but we decided to postpone it: https://github.com/datagouv/hydra/tree/py3.11

Reply:
but, line 102 uses the shorthand `int | None` version, right?

Reply:
Oh indeed, thanks for spotting, fixing that now

Reply:
Fixed